std_msgs/Header header
geometry_msgs/Pose2D goal
int32 remaining_steps
float32 remaining_time

